nabla/config.json

```diff
@@ -36,32 +36,25 @@
       "repository": "nablaContracts",
       "path": "contracts/test/lib/TestableSwapPool.sol"
     },
-    "MockERC20": {
-      "repository": "nablaContracts",
-      "path": "contracts/src/mock/MockERC20.sol"
-    },
     "MockOracle": {
       "repository": "nablaContracts",
       "path": "contracts/src/mock/MockOracle.sol"
     },
     "VendingMachine": {
-      "path": "VendingMachine.contract",
-      "isPrecompiled": true
+      "repository": "nablaCurve",
+      "path": "test/util/VendingMachine.sol"
     }
   },
   "repositories": {
     "nablaContracts": {
       "git": "https://github.com/0xamberhq/contracts.git",
-      "branch": "feature/backstop-pool-coverage-ratio",
+      "branch": "59-incorporate-new-spec",
```

Comment:
This is the branch with the new Nabla version: https://github.com/0xamberhq/contracts/pull/65

Reply:
Yes, I would need to check whether the main branch still works as they added some new oracle related changes.